TileDB.Cloud/RestUtil.cs

```diff
@@ -227,9 +243,23 @@ public static Rest.Model.User GetUser()
             }
             return policyResult.Result;
         }
- 
-        #endregion User
 
+        #endregion User
+ 
+        /// <summary>
+        /// Split a uri into name_space and array_name.
+        /// </summary>
+        /// <param name="array_uri"></param>
+        /// <returns></returns>
+        public static (string name_space, string array_path) SplitUri(string array_uri)
```

Comment:
Can we add a simple test for this function?

Reply:
added a unit test for SplitUri